### ISSUE TITLE
Тексты-презентации партнеров

### ISSUE THREAD

**Issue by VeronikaDed** [VeronikaDed/Discounter#40]
Необходимо привести к единому формату тексты партнеров (отступы, переносы, объем текста):
![1](https://cloud.githubusercontent.com/assets/23305911/23957252/fd912586-09af-11e7-876c-cf59f65398d3.jpg)
![2](https://cloud.githubusercontent.com/assets/23305911/23957253/fdfffba0-09af-11e7-9f8b-e86754efc3e5.jpg)
![3](https://cloud.githubusercontent.com/assets/23305911/23957255/fe08f35e-09af-11e7-881d-53c84c78da8c.jpg)
![4](https://cloud.githubusercontent.com/assets/23305911/23957251/fd799164-09af-11e7-8b9a-9e4b66015ba1.jpg)

**Comment by estpitt** [comment]
Согласен. Нужно обратить на это внимание администратора сайта. Возможно создание рекомендаций по оформлению данного раздела по согласованию с заказчиком.
